gnucash/gnome/gnc-plugin-page-account-tree.c

```diff
@@ -1348,6 +1348,158 @@ gppat_setup_account_selector (GtkBuilder *builder, GtkWidget *dialog,
     return selector;
 }
 
+// This is a helper function for gnc_plugin_page_account_tree_cmd_delete_account
+static void
+gnc_plugin_page_account_tree_cmd_delete_account_int (GtkAction *action, GncPluginPageAccountTree *page, Account* ta,
+                                                    Account* sta, Account* saa, gchar* acct_name, delete_helper_t delete_res)
```

Comment:
I was instructed that helper functions tend to end with _int, but I guess I misunderstood that. I'll fix.

Reply:
I haven't run across any '-int' names elswhere. There are some cases where we have a master static function called `gnc-foo-internal` that takes a bunch of options and public convenience functions `gnc-foo-bar`, `gnc-foo-baz`, etc. that call `gnc-foo-internal` with different option values.  A more common convention is to prefix public function names with the object name, e.g `gnc_numeric_add()` and private implementation functions without. In that convention GObject virtual functions do have the prefix but are still static. You won't find much of that in GnuCash code because there are few if any virtual functions.